src/findinfiles.ts

```diff
@@ -53,6 +54,7 @@ export function createFindInFilesTreeView(
   provider: FindInFilesProvider,
   context: ExtensionContext
 ): TreeView<unknown> {
+  commands.executeCommand("setContext", "metals.showFindInFiles", false);
```

Comment:
It doesn't seem to cause it to disappear when we move to a separate window.

We should also do:
```
const onDidChangeVisibility = treeView.onDidChangeVisibility(async (e) => {
    if (!e.visible)
      commands.executeCommand("setContext", "metals.showFindInFiles", false);
  });
 context.subscriptions.push(onDidChangeVisibility);
```

Reply:
OR actually, maybe we want it to show still :thinking: